src/CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ add_library(configurator_yaml
 target_link_libraries(configurator_yaml
     yaml-cpp::yaml-cpp
     configurator
+    logging_system
```

Comment:
I have suggestion if implementation of configurator should know about logging_system.
I made issue #34 to resolve it.